webapp/publisher/snaps/views.py

```diff
@@ -639,6 +639,18 @@ def redirect_get_register_name():
 @publisher_snaps.route("/register-snap")
 @login_required
 def get_register_name():
+    try:
+        # We don't use the data from this endpoint.
+        # It is mostly used to make sure the user has signed
+        # the terms and conditions.
```

Comment:
this is not really true you use at this line: `available_stores = logic.filter_available_stores(user["stores"])`

Reply:
:+1: